ta/ta.mk

```diff
@@ -121,7 +121,7 @@ $(conf-mk-file-export): $(conf-mk-file)
 	$(q)echo sm := $(sm-$(@)) > $@
 	$(q)echo sm-$(sm-$(@)) := y >> $@
```

Comment:
Correct. `$(conf-mk-file-export)` has to be used consistently. And, in addition, care must be taken so that this variable is actually expanded at the point the definition is included, not when the target is executed. That's because `ta/ta.mk` may be included twice (once to build the 32-bit devkit, and once for the 64-bit one). I'll push an update shortly.